plugins/GSdx/GSState.cpp

```diff
@@ -809,6 +821,16 @@ void GSState::GIFRegHandlerXYZ2(const GIFReg* RESTRICT r)
 
 template<int i> void GSState::ApplyTEX0(GIFRegTEX0& TEX0)
 {
+	// Mask invalid bit before any comparison
+	TEX0.CPSM &= 0xa; // 1010b
+
+	if (PRIM->CTXT != i) {
+		// Context isn't current. Only update the register value.
+		// Handling (real write of the clut buffer) will be done when ctxt will be switched
+		m_env.CTXT[i].TEX0 = (GSVector4i)TEX0;
```

Comment:
If you add the following line (which is also done below) above line 830 it doesn't crash anymore.
```
		if ((TEX0.u32[0] ^ m_env.CTXT[i].TEX0.u32[0]) & 0x3ffffff) // TBP0 TBW PSM
		{
			m_env.CTXT[i].offset.tex = m_mem.GetOffset(TEX0.TBP0, TEX0.TBW, TEX0.PSM);
		}
```